core-services/prow/02_config/generate-boskos.py

```diff
@@ -324,8 +324,8 @@
     },
 }
 
-for i in range(3):
-    for j in range(4):
+for i in range(1,4):
```

Comment:
Changing the range to `1,4` would effectively stop provisioning to `0-0` through `0-4`, is that intended in this PR?

Reply:
Yes, this will reduce the amount of parallel clusters to 6. It's the same amount, which we will have in the intermediate period.

Reply:
As discussed in Slack, I would prefer to leave all clusters engaged until we've introduced all the new clusters, but removing one set would be fine for now given we'll still support a minimally appropriate number of clusters for CI.